src/static/js/utils/process-json.js

```diff
@@ -1,53 +1,5 @@
 'use strict';
 
```

Comment:
Everything renders as expected in IE8 and IE9 but with those JavaScript errors I reported! We should file a bug to fix separately.